share/styles/chords_jazz.xml

```diff
@@ -146,7 +146,7 @@
 
     </font>
 
-  <font class="modifer" family="MuseJazz Text">
+  <font class="modifier" family="MuseJazz Text">
```

Comment:
Looks right to me, but is a code change. @MarcSabatella ?

Reply:
> I checked and saw that there wasn't an actual class named modifer but I did find a whole lot of classes named modifier

Reply:
It's not a class in the C++ sense, but an XML attribute on the font tag, and this typo probably causes some things about chord symbol rendering not to work correctly, so good catch!  Now I just need to make sure whatever change is caused by this (rendering of certain chord symbols in Jazz style) is good, so let me test a little and get back to you.

Reply:
Yes, this actually fixes a bug: the parentheses and comma in C7(b9,b5) do not respond correctly to the "modifier" scaling & offset settings in Format / Style / Chord Symbols.  So this change is not just cosmetic, it is needed in order to address a real issue.

Are there other changes in here like that, or is it mostly non-functional changes?  I do want to make sure this gets fixed, and am trying to decide it is worth submitting a separate PR for just this.